.docker/php/dockerfile

```diff
@@ -16,15 +16,23 @@ RUN apt-get update && apt-get install -y \
 # Clear cache
 RUN apt-get clean && rm -rf /var/lib/apt/lists/*
 
+# Set timezone
+RUN rm /etc/localtime \
+    && ln -s /usr/share/zoneinfo/Africa/Harare /etc/localtime \
+    && "date"
+
 # Install PHP extensions
 RUN docker-php-ext-install pdo_mysql mbstring exif pcntl bcmath gd
 
-# Install Postgre PDO
-RUN apt-get update && apt-get install -y libpq-dev && docker-php-ext-install pdo pdo_pgsql
+# Install MySQL PDO
+RUN apt-get update && apt-get install -y libpq-dev && docker-php-ext-install pdo pdo_mysql
```

Comment:
@scruffycoderofficial Remove this line as `pdo_mysql` has already been taken care of from previous line.